src/model/guild/guild_id.rs

```diff
@@ -303,6 +303,26 @@ impl GuildId {
         http::edit_role(self.0, role_id.into().0, &f(EditRole::default()).0)
     }
 
+    /// Edits the order of [`Role`]s
+    /// Requires the [Mange Roles] permission.
+    ///
+    /// # Examples
+    ///
+    /// Change the order of a role:
+    ///
+    /// ```rust,ignore
+    /// use serenity::model::{GuildId, RoleId};
+    /// GuildId(7).edit_role_position(RoleId(8), 2);
+    /// ```
+    ///
+    /// [`Role`]: struct.Role.html
+    /// [Manage Roles]: permissions/constant.MANAGE_ROLES.html
+    #[inline]
+    pub fn edit_role_position(&self, role_id: u64, position: u64) -> Result<Vec<Role>> {
```

Comment:
Add an `Into<RoleId>` bound and use it instead of `u64` here.
(In order to be consistent with other methods on `GuildId` and other parts of the lib)